tests/integ/test_mxnet_train.py

```diff
@@ -109,6 +110,30 @@ def test_deploy_model_with_tags(mxnet_training_job, sagemaker_session, mxnet_ful
         assert production_variants[0]['InitialInstanceCount'] == 1
 
 
+def test_deploy_model_with_kms_key(mxnet_training_job, sagemaker_session, mxnet_full_version):
```

Comment:
Should we just add this to an existing test? They are already taking so long to run use so much resource.

Reply:
done